lib/config/options/index.ts

```diff
@@ -2034,6 +2034,12 @@ const options: RenovateOptions[] = [
     type: 'boolean',
     default: false,
   },
+  {
+    name: 'automergeCommitMessage',
+    description: 'Optional commit message to use when automerging',
```

Comment:
This short description is too misleading if it's only:
- branch automerge
- merge commit